packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -18,19 +18,19 @@ const addImgOptMiddleware = require('../Utilities/addImgOptMiddleware');
 
 const secureHostWarning = chalk.redBright(
     `  To enable all PWA features and avoid ServiceWorker collisions, PWA Studio
-  highly recommends using the ${chalk.whiteBright(
-      '"provideSecureHost"'
-  )} configuration
-  option of PWADevServer. `
+    highly recommends using the ${chalk.whiteBright(
+        '"provideSecureHost"'
+    )} configuration
+        option of PWADevServer. `
 );
 
 const helpText = `To autogenerate a unique host based on project name
-  and location on disk, simply add:
-    ${chalk.whiteBright('provideSecureHost: true')}
-  to PWADevServer configuration options.
-
-  More options for this feature are described in documentation.
-`;
+        and location on disk, simply add:
+        ${chalk.whiteBright('provideSecureHost: true')}
+        to PWADevServer configuration options.
+        
+        More options for this feature are described in documentation.
+        `;
```

Comment:
changes above are from running `prettier`

Reply:
That's odd...what version of Prettier?

Reply:
i don't have prettier globally on my computer so this is coming from the repo, I can change it back and try it again. Not sure why this triggered in the first place.